app/src/main/java/qds/puck/api/PuckApi.kt

```diff
@@ -0,0 +1,41 @@
+package qds.puck.api
+
+import okhttp3.ResponseBody
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+fun createApi(serverAddress: String): PuckApi {
+    return Retrofit.Builder()
```

Comment:
Are these APIs called anywhere?

If they are we want to implement authentication for them via a login. Meaning we need to have a view for entering the serverAddress.